src/modules/RecordingForm/ConnectionType/index.tsx

```diff
@@ -1,15 +1,47 @@
 import React, { ReactElement } from 'react';
+
+import { Checkbox } from '@aiera/client-sdk/components/Checkbox';
+import { ChangeHandler } from '@aiera/client-sdk/types';
+import { ConnectionType as ConnectionTypeEnum, ConnectionTypeOption } from '@aiera/client-sdk/modules/RecordingForm';
 import './styles.css';
 
-interface ConnectionTypeSharedProps {}
+interface ConnectionTypeSharedProps {
+    connectionType?: ConnectionTypeEnum;
+    connectionTypeOptions: ConnectionTypeOption;
+    onChange: ChangeHandler<ConnectionTypeEnum>;
+}
 
 /** @notExported */
 interface ConnectionTypeUIProps extends ConnectionTypeSharedProps {}
 
-export function ConnectionTypeUI(_props: ConnectionTypeUIProps): ReactElement {
+export function ConnectionTypeUI(props: ConnectionTypeUIProps): ReactElement {
+    const { connectionType, connectionTypeOptions, onChange } = props;
     return (
-        <div className="font-medium py-3 text-gray-400 text-xs tracking-wide uppercase connection-type">
-            <p>Connection Type</p>
+        <div className="py-3 connection-type">
+            <p className="font-semibold mt-2 text-[#C1C7D7] text-xs tracking-widest uppercase">Connection Type</p>
+            <div className="bg-white border border-gray-100 mt-2 rounded shadow-xl">
+                {Object.values(connectionTypeOptions).map((option, idx) => (
+                    <div
+                        className={`${
+                            idx === Object.keys(connectionTypeOptions).length - 1 ? '' : 'border-b'
```

Comment:
Missed this before, but I think you can just use `last:border-b` to have a specific tailwind class only apply to the last child.